File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if (len(s) == len(t)):
            sl = list(s)
            tl = list(t)
            tl.sort()
            sl.sort()
            if  tl == sl:
                return True
            else:
                return False
        else:
            return False

File: Data Structures & Algorithms/anagram-groups/submission-1.java
class Solution {
    public List<List<String>> groupAnagrams(String[] strs) {
        Map<String, List<String>> group = new HashMap<>();
        List<List<String>> ans = new ArrayList();
        for(String s : strs){
            char[] c = s.toCharArray();
            Arrays.sort(c);
            String key = new String(c);
            if(group.containsKey(key)) group.get(key).add(s);
            else {
                List<String> val = new ArrayList();
                val.add(s);
                group.put(key,val);
            }
        }
        ans.addAll(group.values());
        return ans;
    }
}
